src/platform/Linux/bluez/Helper.cpp

```diff
@@ -1718,44 +1721,64 @@ bool BluezUnsubscribeCharacteristic(BLE_CONNECTION_OBJECT apConn)
 
 // ConnectDevice callbacks
 
+using ConnectParams = std::pair<BluezDevice1 *, BluezEndpoint *>;
```

Comment:
not: consider creating a stuct for this. Device/endpoint are much clearer names to work with than first/second. I would prioritize readability over typing speed.

Reply:
ok, will do